tests/defaults.test.js

```diff
@@ -44,28 +44,32 @@ describe("next-on-netlify", () => {
 describe("next-on-netlify", () => {
   test("builds successfully", () => {
     expect(buildOutput).toMatch("Next on Netlify");
-    expect(buildOutput).toMatch("Copying public/ folder to out_publish/");
-    expect(buildOutput).toMatch("Copying static NextJS assets to out_publish/");
     expect(buildOutput).toMatch(
-      "Setting up API endpoints as Netlify Functions in out_functions/"
+      `Copying public${sep} folder to out_publish${sep}`
```

Comment:
File paths are normalized with `join()` in the source code, so those messages print backslashes on Windows.